test/slow/git-process-test.ts

```diff
@@ -153,7 +153,7 @@ echo 'post-check out hook ran'`
 
       Fs.writeFileSync(postCheckoutFile, postCheckoutScript, { encoding: 'utf8', mode: '755' })
 
-      const result = await GitProcess.exec(['checkout', 'master'], testRepoPath)
+      const result = await GitProcess.exec(['checkout', 'main'], testRepoPath)
```

Comment:
Reason this test failed for us locally is because we have updated our global git config with a default initial branch other than `master` which is still Git's default and why it this works when Actions runs it.